src/ast/irbuilderbpf.cpp

```diff
@@ -408,50 +408,85 @@ Value *IRBuilderBPF::CreateStrcmp(Value* val, std::string str, bool inverse) {
 }
 
 Value *IRBuilderBPF::CreateStrcmp(Value* val1, Value* val2, bool inverse) {
+  /*
   // This function compares each character of the two string.
   // It returns true if all are equal and false if any are different
   // strcmp(String val1, String val2)
-  // {
-  //    for (size_t i = 0; i < bpftrace_.strlen_; i++) {
-  //      if (val1[i] != val2[i]) {
-  //        return false;
-  //      }
-  //    }
-  //    return true;
-  // }
-
+     {
+        for (size_t i = 0; i < bpftrace_.strlen_; i++) {
```

Comment:
Nitpick, but coud you fix the indentation and bracketing for this pseudo code? Sorry!

Reply:
No problem